### ISSUE TITLE
[Bug]- Installing rancher 2.X using nerdctl 

### ISSUE THREAD

**Issue by evertonlperes** [rancher-sandbox/rancher-desktop#970]
### Rancher Desktop Version

0.6.1

### What operating system are you using?

Other Linux

### Operating System / Build Version

openSUSE Leap 15.3 / 5.3.18-59.27

### What CPU architecture are you using?

x64

### Windows User Only

N/A

### Actual Behavior

Installing Rancher 2.X on Rancher Desktop using `nerdctl` does not work properly.
The container crashes after many attempts to find the CRDs to be available and the `k3s` exited with error code 255.

### Steps to Reproduce

1. Install RD latest release (0.6.1)
2. Wait until the app be ready
3. Install Rancher 2.x using: `nerdctl run --privileged -d --restart=no -p 8080:80 -p 8443:443 rancher/rancher`
4. Get the container ID: `nerdctl ps`
5. Watch the logs: `nerdctl logs -f [ContainerID]
6. Should see errors/info related to the missing CRDs and cannot load https://localhost.8443

### Result

Cannot open Rancher Dashboard on https://localhost.8443

### Expected Behavior

Should install Rancher 2.x properly using `nerdctl`

### Additional Information

TLDR from container log:

```bash
2021/11/16 22:20:55 [INFO] Waiting for CRD multiclusterapps.management.cattle.io to become available
2021/11/16 22:20:56 [INFO] Done waiting for CRD multiclusterapps.management.cattle.io to become available
2021/11/16 22:20:56 [INFO] Waiting for CRD cisconfigs.management.cattle.io to become available
2021/11/16 22:20:57 [INFO] Done waiting for CRD cisconfigs.management.cattle.io to become available
2021/11/16 22:20:57 [INFO] Waiting for CRD templates.management.cattle.io to become available
2021/11/16 22:20:59 [INFO] Done waiting for CRD templates.management.cattle.io to become available
2021/11/16 22:20:59 [INFO] Waiting for CRD templatecontents.management.cattle.io to become available
2021/11/16 22:21:00 [INFO] Done waiting for CRD templatecontents.management.cattle.io to become available
2021/11/16 22:21:00 [INFO] Waiting for CRD globaldnses.management.cattle.io to become available
2021/11/16 22:21:01 [INFO] Done waiting for CRD globaldnses.management.cattle.io to become available
2021/11/16 22:21:01 [INFO] Waiting for CRD globaldnsproviders.management.cattle.io to become available
2021/11/16 22:21:02 [INFO] Done waiting for CRD globaldnsproviders.management.cattle.io to become available
2021/11/16 22:22:16 [FATAL] k3s exited with: exit status 255
```

There's a strange behaviour on the builder pod during the install where the buildkit just restart with the following messages:

Logs from `builder kit` container:
```bash

time="2021-11-16T23:20:46Z" level=warning msg="using host network as the default"
time="2021-11-16T23:20:46Z" level=info msg="found worker \"qalm7ct6711104zxlpghg105w\", labels=map[org.mobyproject.buildkit.worker.executor:containerd org.mobyproject.buildkit.worker.hostname:lima-rancher-desktop org.mobyproject.buildkit.worker.snapshotter:overlayfs], platforms=[linux/amd64 linux/arm64 linux/386]"
time="2021-11-16T23:20:46Z" level=info msg="found 1 workers, default=\"qalm7ct6711104zxlpghg105w\""
time="2021-11-16T23:20:46Z" level=warning msg="currently, only the default worker can be used."
time="2021-11-16T23:20:47Z" level=warning msg="TLS is disabled for unix:///run/buildkit/buildkitd.sock"
time="2021-11-16T23:20:47Z" level=info msg="running server on /run/buildkit/buildkitd.sock"
time="2021-11-16T23:20:47Z" level=info msg="running server on [::]:1234"
time="2021-11-16T23:25:29Z" level=info msg="stopping server"
buildkitd: context canceled
stream closed
```

Events from `builder` pod:
```bash
  Normal   Created    16m                   kubelet, lima-rancher-desktop  Created container rshared-containerd
  Normal   Started    16m                   kubelet, lima-rancher-desktop  Started container rshared-containerd
  Normal   Pulled     16m                   kubelet, lima-rancher-desktop  Container image "docker.io/moby/buildkit:v0.8.3" already present on machine
  Normal   Created    16m                   kubelet, lima-rancher-desktop  Created container buildkit
  Normal   Started    15m                   kubelet, lima-rancher-desktop  Started container buildkit
  Normal   Pulling    15m                   kubelet, lima-rancher-desktop  Pulling image "rancher/kim:v0.1.0-beta.6"
  Normal   Pulled     15m                   kubelet, lima-rancher-desktop  Successfully pulled image "rancher/kim:v0.1.0-beta.6" in 14.439206306s
  Normal   Created    15m                   kubelet, lima-rancher-desktop  Created container agent
  Normal   Started    15m                   kubelet, lima-rancher-desktop  Started container agent
  Warning  Unhealthy  7m30s (x17 over 14m)  kubelet, lima-rancher-desktop  Liveness probe failed:
  Warning  Unhealthy  2m29s (x27 over 15m)  kubelet, lima-rancher-desktop  Readiness probe failed:
```
﻿﻿﻿﻿﻿
**Logs**

[nerdctl_deploy_logs.zip](https://github.com/rancher-sandbox/rancher-desktop/files/7550539/nerdctl_deploy_logs.zip)

**Comment by evertonlperes** [comment]
Running the installation under a really slow environment (e.g.: virtualbox), makes the installation process quite goofy and not trusty.
Closing it for now.
